src/axom/quest/examples/CMakeLists.txt

```diff
@@ -60,22 +60,25 @@ if(ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
 
         # Non-zero empty-rank probability tests domain underloading case
         set(_emptyRankProbs 0.0 0.5)
+        set(_meshes "mdmesh.2x1" "mdmesh.2x3")
```

Comment:
Should we also test this with a query mesh that is not multi-domain to ensure the conversion logic works?